src/include/duckdb/common/types.hpp

```diff
@@ -308,10 +310,10 @@ enum class PhysicalType : uint8_t {
 	/// Like LIST, but with 64-bit offsets
 	LARGE_LIST = 33,
 
-	// DuckDB Extensions
+	/// DuckDB Extensions
 	VARCHAR = 200, // our own string representation, different from STRING and LARGE_STRING above
 	INT128 = 204, // 128-bit integers
-
+	UNKNOWN = 205, // Unknown physical type of user defined types
```

Comment:
Is this supposed to be the same as `BIT` ?

Reply:
I guess this was a merge issue